os/drivers/audio/ndp120_voice.h

```diff
@@ -78,6 +78,7 @@ struct ndp120_dev_s {
 	uint32_t extract_size;
 	bool extclk_inuse;
 	volatile bool alive;
+	bool keyword_correction;
```

Comment:
on what basis the keyword_correction is enabled and disabled? Do we need to provide this control also to the application?